components/cam/src/physics/cam/micro_p3_utils.F90

```diff
@@ -195,7 +214,7 @@ subroutine micro_p3_utils_init(cpair,rair,rh2o,rhoh2o,mwh2o,mwdry,gravit,latvap,
 
     cons1 = piov6*rhow
     cons2 = 4._rtype*piov3*rhow
-    cons3 = 1._rtype/(cons2*(25.e-6_rtype)**3)
+    cons3 = 1._rtype/(cons2*1.562500000000000d-14)  ! 1._rtype/(cons2*bfb_pow(25.e-6_rtype,3.0_rtype))
```

Comment:
It looks like you aren't actually using the bfb math layer, so you shouldn't need the cpp stuff you added to the top of this file.